tools/geojson_simplifier/simplify_test.py

```diff
@@ -0,0 +1,88 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Tests the import_data.py script.
+
+    Typical usage:
+    python3 simplify_test.py
+"""
+import unittest
+import geojson
+import simplify
+import os
+
+TEST_DATA_DIR = "test-data"
+
+
+class GeojsonSimplifierTest(unittest.TestCase):
+    def test_california(self):
+        simplifier = simplify.GeojsonSimplifier()
+        simplifier.read_geojson(os.path.join(TEST_DATA_DIR,
+                                             'california.geojson'))
+        simplifier.simplify()
+        result = simplifier.geojson
+        file = os.path.join(TEST_DATA_DIR, 'california-simple.geojson')
+        with open(file, 'r') as f:
+            expected_result = geojson.load(f)
+        self.assertDictEqual(result, expected_result)
+
+    def test_alabama(self):
```

Comment:
These two tests contain identical code, just different comparison data. Could you refactor some code out? What's important to do after that is to ensure that the message that gets printed out on failure tells you which file caused the issue (otherwise it's hard to know which case failed).